### PR TITLE
James/ar updates

### PR DESCRIPTION
**Pull request by JamesMBallard**
Updates for AR changes coming with 200.2.

I'll add some inline comments.

### CODE REVIEW

File: augmentedreality/README.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ Android 7.0 64-bit (armv8) is the minimum requirement. 32-bit (armv7) is not sup
 
 The latest ArcGIS Maps SDK for Qt is required to use this toolkit component.
 
+The system requirements for the ArcGIS Maps SDK for Qt takes precendent over the minimum versions mentioned here.
```
Comment (JamesMBallard) [inline augmentedreality/README.md:65]: A few lines above this one we mention some very old iOS and Android OSes. Probably the first ones to ever support AR, which is much older than our system requirements mention now. 

I didn't want to change those and have more maintenance, so hopefully this statement will make it clear which document wins.

cc @ldanzinger

File: uitools/examples/cpp_quick/src/main.cpp [hunk 2]
```diff
@@ -35,8 +35,6 @@
 //------------------------------------------------------------------------------
 int main(int argc, char* argv[])
 {
-  // Enforce OpenGL
-  qputenv("QSG_RHI_BACKEND", "opengl");
```
Comment (JamesMBallard) [inline uitools/examples/cpp_quick/src/main.cpp:39]: This is unrelated changes I missed in an earlier round of changes. We no longer need to force OpenGL for the toolkit examples that are not AR-specific.

File: augmentedreality/README.md (hunk 1 above)
Comment (tanneryould) [inline augmentedreality/README.md:65]: Small grammar and spelling correction.

```suggestion
The system requirements for the ArcGIS Maps SDK for Qt take precedent over the minimum versions mentioned here.
```

**Review by GuillaumeBelz — APPROVED** [review]

**Review by tanneryould — APPROVED** [review]
LGTM!
